### PR TITLE
Move cards to local storage

### PR DESCRIPTION
**Pull request by FSchubert92**

### CODE REVIEW

File: src/Create/NewCardForm.js [hunk 1]
```diff
@@ -119,7 +122,7 @@ export default function CreateCard(props) {
   const locationLength = data.location.length > 0
```
Comment (lcwalther) [inline src/Create/NewCardForm.js]: why > 0? does it only save the variable when true?

File: src/Create/NewCardForm.js [hunk 2]
```diff
@@ -166,69 +169,73 @@ export default function CreateCard(props) {
...
+          <h3>Image</h3>
+          <input
+            onChange={onFileChange}
+            type="file"
+            required
+            accept="image/x-png,image/gif,image/jpeg"
+          />
+        </div>
+        <ButtonWrapper>
+          <BackButton to="/">X</BackButton>
```
Comment (lcwalther) [inline src/Create/NewCardForm.js]: If it is possible to access this site from another route but "/" maybe a history back function would also work here

File: src/Home/Home.js [hunk 3]
```diff
@@ -22,23 +22,39 @@ const CreateButton = styled(NavLink)`
...
+          <NoCardsToShow>
+            Sorry, there are no postcards to show TT <br />
+            Why don't you start adding one?
+          </NoCardsToShow>
+        ) : (
+          cards.map(card => (
+            <Card
+              date={dayjs(card.date).format('dddd  DD MMMM YYYY ')}
+              location={card.location}
+              temperatur={card.temperatur}
```
Comment (lcwalther) [inline src/Home/Home.js]: Should it be card.temperatur**e**?

File: src/Home/Home.js [hunk 4]
```diff
@@ -22,23 +22,39 @@ const CreateButton = styled(NavLink)`
...
+              picture={card.picture}
+              summary={card.summary}
+              iAte={card.food}
+              itTasted={card.taste}
+              key={card.id}
+            />
+          ))
+        )}
       </CardContainer>
       <CreateButton to="/create">
```
Comment (lcwalther) [inline src/Home/Home.js]: Nice way to link the button!

File: src/services.js [hunk 5]
```diff
@@ -19,7 +41,20 @@ export function deleteCardFromServer(card) {
 }
 
 export function getCardsFromStorage() {
-  return getFromStorage('cards') || []
+  return (
+    getFromStorage('cards') || [
+      {
+        date: '',
+        location: '',
+        temperatur: '20C°',
```
Comment (lcwalther) [inline src/services.js]: temperature? see above

File: src/services.js [hunk 6]
```diff
@@ -1,9 +1,31 @@
...
+  const formData = new FormData()
+  formData.append('file', picture)
+  formData.append('upload_preset', PRESET)
+
+  return axios.post(url, formData, {
+    headers: {
+      'Content-type': 'multipart/form-data',
+    },
+  })
+  // .then(onImageSave)
```
Comment (lcwalther) [inline src/services.js]: Why did you exclude the catch error?

File: src/Create/NewCardForm.js (hunk 1 above)
Comment (FSchubert92) [reply src/Create/NewCardForm.js]: It should return either 'true' or 'false' When something is put in, the length is > 0 and therefore it returns 'true'

File: src/Create/NewCardForm.js (hunk 2 above)
Comment (FSchubert92) [reply src/Create/NewCardForm.js]: That is a good idea!

File: src/Home/Home.js (hunk 3 above)
Comment (FSchubert92) [reply src/Home/Home.js]: it should :(  but I might just keep it like this, before I change every thing :D

**Pull request closed without merging**